src/models/session/logic.test.ts

```diff
@@ -112,9 +130,10 @@ describe("findNextNodeOfType", () => {
     });
     expect(nextNode).toEqual({
       id: "secondQuestion",
+      rootNodeId: "secondQuestion",
       parentId: "firstQuestion",
       type: ComponentType.Question,
-      edges: ["secondAnswer"],
+      edges: ["Q2Answer1", "Q2Answer2", "Q2Answer3"],
```

Comment:
The test flow has been updated to include more branches (no functional change here).